client/windows/GUIClasses.cpp

```diff
@@ -615,6 +655,32 @@ void CTavernWindow::HeroPortrait::hover(bool on)
 		GH.statusbar()->clear();
 }
 
+CTavernWindow::HeroSelector::HeroSelector(std::map<HeroTypeID, CGHeroInstance*> InviteableHeroes, std::function<void(CGHeroInstance*)> OnChoose)
```

Comment:
Can't reuse starting hero selector window?

Reply:
Possible. But not that easy. Would need some refactoring, as hero selector window from main menu uses some other internal classes from mainmenu. Don't want to do this now.